docs/source/starter/new-project.rst

```diff
@@ -61,8 +102,33 @@ You could also use conda environments
     conda activate my_env
     pip install pytorch-lightning
 
+Install future patch release from the source
+
+.. code-block:: bash
+
+    pip install git+https://github.com/PytorchLightning/pytorch-lightning.git@release/1.5.x --upgrade
+
+Install nightly from the source (no guarantees)
```

Comment:
not convinced this should be part of the quick start guide, as 99% users will not want to quick start with nightly (why would they). This is distracting, as the user is has to make a choice.

Reply:
just wanted to introduce a section that tells all possible ways of installation. Pytorch mentions nightly too on their website: https://pytorch.org/

but you are right, maybe if we agree on the introduction page, this can be moved completely out of this one and we can just mention the installation guide for the published one.

Reply:
yeah but their nightly is not bare master but still need to tested and compiled which is not the same as you suggest user to use PL sourcecode :]